src/python/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Flask==0.12.2
+Flask-Compress==1.4.0
+ujson==1.35
```

Comment:
Is csv library a core one or should be added here as well?

Reply:
`csv` package is part of the standard library https://docs.python.org/3/library/csv.html that's why is not included